packages/inspection-capture-web/src/PhotoCapture/PhotoCaptureHUD/PhotoCaptureHUDPreviewSight/AddDamageButton/AddDamageButton.tsx

```diff
@@ -10,18 +10,26 @@ export interface AddDamageButtonProps {
    * Callback called when the user presses the button.
    */
   onAddDamage?: () => void;
+  /**
+   * Boolean indicating whether the Add Damage feature is disabled. If disabled, the `Add Damage` button will be hidden.
+   *
+   * @default false
+   */
+  disableAddDamage?: boolean;
 }
 
 /**
  * Custom button displayed in the PhotoCapture Camera HUD that allows user to enter add damage mode.
  */
-export function AddDamageButton({ onAddDamage }: AddDamageButtonProps) {
+export function AddDamageButton({ onAddDamage, disableAddDamage }: AddDamageButtonProps) {
   const { t } = useTranslation();
   const primaryColor = usePhotoCaptureHUDButtonBackground();
 
   return (
     <Button
       icon='add'
+      style={{ visibility: disableAddDamage ? 'hidden' : 'visible' }}
```

Comment:
either : style.ts or function like "getButtonStyle" or hook "useStyle" (i think function in the same file is fine)